src/components/AttachmentModal.js

```diff
@@ -326,6 +348,32 @@ function AttachmentModal(props) {
 
     const sourceForAttachmentView = props.source || source;
 
+    const threeDotMenuItems = [
```

Comment:
The `HeaderWithBackButton` prop name is `threeDotsMenuItems` so can we change this to:

```suggestion
    const threeDotsMenuItems = [
```